lib/queries/github_user_info.graphql

```diff
@@ -0,0 +1,6 @@
+query ($user: String!) {
+    user(login: $user) {
```

Comment:
our other graphql file uses two space indent.

if you think four is better, also create an issue to re-indent the existing file, and we'll make it a newcomer task

Reply:
I should have used two space indent. Blame my editor :P